src/jsMain/js/simulator/windows/ShaderEditorWindow/ShaderEditorWindow.jsx

```diff
@@ -97,7 +120,40 @@ const ShowEditorWindow = (props) => {
   });
 
   return (
-    <div className={styles.showEditorWindow} ref={windowRootEl}>
+    <div className={styles.shaderEditorWindow} ref={windowRootEl}>
+      <Tabs
+          variant="scrollable"
+          onChange={handleTabChange}
+      >
+        {openShaders.map((shaderInfo) => {
+          return <Tab
+              label={shaderInfo.name}
+          />
+        })}
+
+        <div>
+          <Button
+              aria-controls="simple-menu"
+              aria-haspopup="true"
+              variant="contained"
+              onClick={handleMenuButton}>
+            …
```

Comment:
Is there a reason to have this line?

Not entirely sure what
```
…
```
is supposed to be doing on this line?

Not to be confused with the [... spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax)

Reply:
I was going for a vertical ellipsis, like a menu dropdown. All this code is really just FPO, I'd love help replacing it with better UI and code design.

Reply:
Oh sorry, I totally misread that line and didn't see the closing `>` before the `...`.